src/mir/dataflow.md

```diff
@@ -217,6 +217,26 @@ the example below:
 
 ![A graphviz diagram for a dataflow analysis](../img/dataflow-graphviz-example.png)
 
+### Region Constraint Graphs and Their Strongly Connected Components
+
+![A graph showing a small number of regions with their outlives relations](../img/region-graphviz.png)
+
+With `-Z dump-mir-graphviz=yes`, you will also get Graphviz files for the outlives constraints
```

Comment:
I didn't know what either of these did, but I'll figure it out and add them

Reply:
Update: that seems like too much for one PR, I'll do something more minimal and come back for them as I use them

Reply:
I couldn't find `-Zdump-mir=nll` documented anywhere, what does it mean?

Reply:
Does this mean if I have `fn nll()` I can't dump its MIR because the `mir` value is special?

Reply:
I don't believe so, no, it's probably just a bunch of "check if it applies" tests. Such collisions would already exist with `-Zdump-mir=all` otherwise.

Reply:
https://github.com/rust-lang/rustc-dev-guide/blob/master/src/mir/debugging.md also describes the complicated micro syntax that `-Zdump-mir` has